misc/dataframeclustering/readme.md

```diff
@@ -0,0 +1,3 @@
+# DataFrameClustering Module
+
+Description of the DataFrameClustering module
```

Comment:
Suggestion:
This module provides the functionality for clustering the rows of a DataFrame. Supported clustering methods are _k-means_, _DBSCAN_, _agglomerative_, and _spectral_ clustering.
 The clustering is performed in python using the following modules: `...`.